TemplateMethod/EmailVerification.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Epam.NetMentoring.Adapter.TemplateMethod
+{
+    public abstract class EmailVerification
```

Comment:
Better to use a more self descriptive name - `EmailValidator` or `EmailVerificator`

Reply:
ok